job-dsls/jobs/DailyBuild_pipeline.groovy

```diff
@@ -135,8 +135,18 @@ pipeline {
             steps {
                 cleanWs()
             }
-        }                                 
-    }
+        }                                    
+    }
+    post {
+        failure{
+            script {
+                currentBuild.result = 'FAILURE\'
+            }            
+            emailext body: 'status of daily build #${BUILD_NUMBER} (${baseBranch} branch) was: ' + "${currentBuild.currentResult}" +  '\\n' +
```

Comment:
Sorry for my ignorance, just curious if `currentBuild.result` or `currentBuild.currentResult` is correct? Have you tested it?

Reply:
sure - that is tested! On my local it worked. This is also defined in the other mailers and the bsig list received the mails. Strange to say - but currentBuild.result didn't fetch the expected value - so I did  currentResult

Reply:
Ok, cool! Thanks!